module-sys/SystemManager/LogSentinel.cpp

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2017-2023, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#include <SystemManager/LogSentinel.hpp>
+#include <log/log.hpp>
+
+namespace sys
+{
+    namespace
+    {
+        constexpr std::uint32_t holdMutexTimeoutMs{100000};
+        constexpr auto sentinelName{"LoggerSentinel"};
+        constexpr auto holdFrequencyReason{"LogDump"};
+    } // namespace
+
+    LogSentinel::LogSentinel(bsp::CpuFrequencyMHz frequencyToLock) : requestedFrequency(frequencyToLock)
+    {
+        mutex = xSemaphoreCreateMutex();
+        if (mutex == NULL) {
```

Comment:
Try to use `nullptr` ;)

Reply:
![Zrzut ekranu z 2023-10-25 10-57-40](https://github.com/mudita/MuditaOS/assets/70628239/5ea0b011-5b7d-429c-8743-9d030d545de9)
I follow the guidelines from the FreeRTOS documentation